Mathlib/Topology/UniformSpace/UniformEmbedding.lean

```diff
@@ -130,70 +130,107 @@ protected theorem UniformInducing.injective [T0Space α] {f : α → β} (h : Un
 /-- A map `f : α → β` between uniform spaces is a *uniform embedding* if it is uniform inducing and
 injective. If `α` is a separated space, then the latter assumption follows from the former. -/
 @[mk_iff]
-structure UniformEmbedding (f : α → β) extends UniformInducing f : Prop where
+structure IsUniformEmbedding (f : α → β) extends UniformInducing f : Prop where
```

Comment:
deprecation missing

Reply:
@adomani, all the deprecations were generated using your script, so you want to see which were missing to fix the script. Looks like it missed `structure` and `abbrev`

Reply:
I think that the deprecation script was a precursor to the decl_diff script and is missing all the future improvements that were added to decl_diff.

I'll see if I get a chance to modify the decl_diff script, or at least try to merge it into the deprecation one, though the term has started and I have less time available now.

Reply:
@YaelDillies, I adapted the script and ran it from your *first* commit.  Besides what I think were the modifications that you also had there, it added a few more, such as

```diff
@@ -104,6 +104,9 @@ abbrev IsUniformEmbedding.comapMetricSpace {α β} [UniformSpace α] [m : Metric
     (h : IsUniformEmbedding f) : MetricSpace α :=
   .replaceUniformity (.induced f h.inj m) h.comap_uniformity.symm
 
+@[deprecated (since := "2024-10-04")]
+alias UniformEmbedding.comapMetricSpace := IsUniformEmbedding.comapMetricSpace
+

@@ -222,6 +222,9 @@ noncomputable def ofIsUniformEmbedding (f : α → β) (hf : IsUniformEmbedding
     exact uniformContinuous_subtype_val
   toEquiv := Equiv.ofInjective f hf.inj
 
+@[deprecated (since := "2024-10-04")]
+alias ofUniformEmbedding := ofIsUniformEmbedding
+

@@ -134,12 +134,15 @@ structure IsUniformEmbedding (f : α → β) extends UniformInducing f : Prop wh
   /-- A uniform embedding is injective. -/
   inj : Function.Injective f
 
+@[deprecated (since := "2024-10-04")]
+alias UniformEmbedding := IsUniformEmbedding
+

####  not sure about this one in `Mathlib/Topology/UniformSpace/UniformEmbedding.lean` ~270
+@[deprecated (since := "2024-10-04")]
+alias UniformEmbedding.denseEmbedding := IsUniformEmbedding.denseEmbedding
+
```

I imagine that these are the ones that were later flagged by Jireh.  The last one seems strange.  There were others as well, but they looked more like later changes to master.

Reply:
Interesting! Please post the updated script on Zulip